common/src/main/java/fi/hel/verkkokauppa/common/configuration/ExperienceUrls.java

```diff
@@ -9,4 +9,7 @@
 public class ExperienceUrls {
     @Value("${order.experience.url:http://order-experience-api:8080/v1/order/}")
     private String orderExperienceUrl;
+
+    @Value("${payment.experience.url:http://payment-experience-api:8080/v1/payment/}")
```

Comment:
You need to make this url to be real experience api url, because paytrail cant call platta internal urls. You can do it easily with, adding property to application.properties